### PR TITLE
feat(graphics): add a rogue build guide

### PR DESCRIPTION
**Pull request by StaticRocket**
Add a short guide to go over the different pieces of the proprietary and open source stack, what they do, and how to build them.

### CODE REVIEW

**Review by cshilwant — DISMISSED** [review]

File: source/linux/Foundational_Components/Graphics/Rogue/Build_Guide.rst [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+   - ``BUILD`` -- Options: ``release`` or ``debug``
+   - ``PVR_BUILD_DIR`` -- Options: ``<platform>_linux``, for example ``am62_linux``
+   - ``WINDOW_SYSTEM`` -- Options: ``lws-generic``, ``xorg``, ``wayland``, ...
+
+These variables must be set for the build. To check for updated build variables
+please see the ``EXTRA_OEMAKE`` variable in the `ti-img-rogue-driver recipe in
+meta-ti`_. This will always contain the latest required variables to compile the
+driver.
+
+.. _ti-img-rogue-driver recipe in meta-ti: https://git.ti.com/cgit/arago-project/meta-ti/tree/meta-ti-bsp/recipes-bsp/powervr-drivers/ti-img-rogue-driver_24.1.6554834.bb
```
Comment (cshilwant) [inline source/linux/Foundational_Components/Graphics/Rogue/Build_Guide.rst:61]: Same comment as https://github.com/TexasInstruments/processor-sdk-doc/pull/57#discussion_r1867152237

Comment (StaticRocket) [reply source/linux/Foundational_Components/Graphics/Rogue/Build_Guide.rst:61]: This one will change eventually, but I intent to point this to the latest version on each release.

Comment (StaticRocket) [reply source/linux/Foundational_Components/Graphics/Rogue/Build_Guide.rst:61]: The idea being, this section should be checked for updates upon each release anyway.

Comment (cshilwant) [reply source/linux/Foundational_Components/Graphics/Rogue/Build_Guide.rst:61]: Great, that you are planning to keep it updated in each release.

My concern was on SDK docs of older version (they are older to us but still relevant to users) which would eventually point to broken link once the recipe name has an update.

Better to use tag as well in such links?

Probably - https://git.ti.com/cgit/arago-project/meta-ti/tree/meta-ti-bsp/recipes-bsp/powervr-drivers/ti-img-rogue-driver_24.1.6554834.bb?h=10.01.09

Comment (StaticRocket) [reply source/linux/Foundational_Components/Graphics/Rogue/Build_Guide.rst:61]: Ah, yeah. Fair point.

**Review by Krunalb11 — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]

**Comment by cshilwant** [comment]
@DarrenEtheridge @praneethbajjuri for reviews

**Review by praneethbajjuri — APPROVED** [review]

**Pull request merged**